gcloud/dns/changes.py

```diff
@@ -221,7 +221,8 @@ def create(self, client=None):
         self._set_properties(api_response)
 
     def exists(self, client=None):
-        """API call:  test for the existence of the change set via a GET request
+        """API call:  test for the existence of the change set via a GET
+        request.
```

Comment:
@jonparrott ISTM that breaking with the PEP is just wrong: the "single line summary" discipline *works* as a way to focus intent.  When I find myself creeping up to the limit, finding a way to shorten that line (maybe by moving some content down to a more "narrative" paragraph) always improves it.